packages/experiments/src/components/Tile/Tile.tsx

```diff
@@ -293,14 +293,14 @@ export class Tile extends BaseComponent<ITileProps, ITileState> {
     return (
       <span key="nameplate" className={css('ms-Tile-nameplate', TileStyles.nameplate)}>
         {name ? (
-          <span id={this._nameId} className={css('ms-Tile-name', TileStyles.name)}>
+          <div id={this._nameId} className={css('ms-Tile-name', TileStyles.name)}>
```

Comment:
Everything in a `Tile`'s content is nested within an `a`, which according to W3C spec should never have `<div>` nested inside it, which is why everything is a `span`. You can use CSS to set `display: block` though.

Reply:
Ah okay, I'll change it to a `span`